### ISSUE TITLE
Add alternative installation instructions

### ISSUE THREAD

**Issue by MatthijsBlom** [JuliaLang/juliaup#175]
**Linux & Mac:** I couldn't find specific instructions, but I'm guessing they exist, as it sounds like Juliaup should work on these platforms. 

**Windows:** For some, installing from the Store may not be an option. How should these people install Juliaup?

It would be nice if the README listed these installation instructions.

**Comment by davidanthoff** [comment]
The only option that is ready at this point is the Windows Store option. We are getting closer with Linux and MacOS support, but at this point it is not ready for general consumption or feedback, so for now we should wait with install instructions in the README until things are ready :) But then, yes, agreed, we need to add them! I'll keep this issue open to track that.

**Comment by DNF2** [comment]
Related to this: I understand that juliaup should update itself. Does this, too, rely on a connection with the Windows Store? Getting the Windows Store to work is very hard, so after the first install, it would be good to not rely on juliaup communicating with the store.

**Comment by davidanthoff** [comment]
Yes, the version from the Windows Store gets updated by the Windows Store. It solves _a lot_ of problems as doing background updates is not trivial...

My current plan is to next figure out automatic self-update for Linux and Mac, and then come back to Windows and see whether we can improve the situation for folks where the Windows Store is in some form blocked.

**Comment by MatthijsBlom** [comment]
> [&hellip;] doing background updates is not trivial...

What exactly do you mean by 'background updates'?

Off the top of my head, Powershell (`pwsh`) warns that an upgrade is available on start, and `pip` likewise warns at least when used to install a package (and maybe in more cases). Both require the user to initiate the upgrade &ndash; typically through manual download in the case of `pwsh`, but a simple one-liner in the case of `pip` (and `rustup`, and `stack`, and&hellip;). I'm fine with the latter strategy.

**Comment by DNF2** [comment]
Does the new-ish Windows package manager, winget, change the picture at all, or is that just a CLI for the Store app?

**Comment by MatthijsBlom** [comment]
> [&hellip;] or is that just a CLI for the Store app?

It is not, but it can function as one:

```
PS ~ > winget search julia
Name                 Id              Version Source
-----------------------------------------------------
Julia                9NJNWW8PVKMN    Unknown msstore
Julian Date Selector 9NSGP4VDNW0R    Unknown msstore
Julia                Julialang.Julia 1.6.2   winget
```

I tried to circumvent the Store by using `winget`, but it doesn't work: installing 9NJNWW8PVKMN still requires logging in.

**Comment by ItzLevvie** [comment]
> I tried to circumvent the Store by using `winget`, but it doesn't work: installing 9NJNWW8PVKMN still requires logging in.

Blocked by https://github.com/microsoft/winget-cli/issues/1585#issuecomment-974509235.

**Comment by fingolfin** [comment]
This has been completed, I'd say. The remaining caveat -- limitations and reliability of non-Store Windows installation -- is already tracked by #343.
